meta/kwoptarg.rb

```diff
@@ -1,11 +1,11 @@
 Mutest::Meta::Example.add :kwarg do
-  source 'def foo(bar: baz); end'
+  source 'def foo(bar: {}); end'
```

Comment:
Yeah, I considered that but since `baz` is unaltered it didn't seem like it mattered. I suppose it _could_ though. I am currently doing a full run on the entire codebase on my laptop so I can confirm / fix that if it is a problem later. Or I can add another example if you prefer--it's just more duplication.